integration-tests/rest-client-reactive/src/test/java/io/quarkus/it/rest/client/InjectMockTest.java

```diff
@@ -14,6 +15,7 @@
 import io.restassured.RestAssured;
 
 @QuarkusTest
+@Order(100) // used in order to make sure this is run before ShouldNotUseMockTest
```

Comment:
@geoand Will this annotation be respected without further configuration? Wouldn't this require also setting `junit.quarkus.orderer.secondary-orderer`?

Reply:
Before the change in the PR, this  test would fail, with the change it passes.

But let's ask @famod to be sure.

Reply:
PS: We could also set this property in build-parent, so that `@Order` works in all modules, without forcing it on Quarkus end users.

Reply:
Thanks, I'll update soon

Reply:
I added the system property to the integration-tests parent